distributed/hdfs.py

```diff
@@ -46,6 +49,8 @@ def read_bytes(fn, executor=None, hdfs=None, lazy=False, delimiter=None, **hdfs_
     blocks = get_block_locations(hdfs, fn)
     filenames = [d['filename'] for d in blocks]
     offsets = [d['offset'] for d in blocks]
+    if not_zero:
+        offsets = [max([o, 1]) for o in offsets]
```

Comment:
Hrm, I guess that this would be helpful in both the csv and avro cases, giving this some weight.